lib/libc/atof.c

```diff
@@ -0,0 +1,59 @@
+/* 
+* Copyright (C) 2014, Galois, Inc.
+* This sotware is distributed under a standard, three-clause BSD license.
+* Please see the file LICENSE, distributed with this software, for specific
+* terms and conditions.
+*/
```

Comment:
Can you replace this with the actual full body of the LICENSE from the repo it came with?

Reply:
i did